### PR TITLE
5577 | schedule error states 

### PR DESCRIPTION
**Pull request by zoranikolic**

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Comment by votemike** [comment]
Should the page also be shown if the service isn't found? Or is the exception there fine?

**Comment by zoranikolic** [comment]
@votemike 404/500 service error pages have been moved to new ticket for next release

**Comment by votemike** [comment]
Does the link to the start/end date need to be styled?

**Comment by votemike** [comment]
Also, what happens if `$servicesInNetwork` is falsey?

File: app/Resources/views/schedules/no_schedule.html.twig [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{% extends 'base_ds2013.html.twig' %}
+
+{% block body %}
+
+    <p class="noschedule">
+        {% if date < service.startDate %}
```
Comment (votemike) [inline app/Resources/views/schedules/no_schedule.html.twig]: What is a service doesn't have a startDate?

File: app/Resources/views/schedules/no_schedule.html.twig [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+            {{ tr('schedule_begins_on', {'%1':'<a href="' ~ path('schedules_by_day', {'pid': service.pid, 'date': service.startDate|date('Y-m-d')}) ~ '">' ~ service.startDate|date("l j F Y") ~ '</a>'})|raw }}
+        {% elseif date > service.endDate %}
```
Comment (votemike) [inline app/Resources/views/schedules/no_schedule.html.twig]: What is a service doesn't have a endDate?

File: app/Resources/views/schedules/no_schedule.html.twig (hunk 1 above)
Comment (BPScott) [inline app/Resources/views/schedules/no_schedule.html.twig]: If a service does not have a start date it should be considered available, so `date and date < service.startDate`

File: app/Resources/views/schedules/no_schedule.html.twig (hunk 2 above)
Comment (BPScott) [inline app/Resources/views/schedules/no_schedule.html.twig]: If a service does not have a end date it should be considered available, so `date and date > service.endDate`

**Pull request merged**